### ISSUE TITLE
If updating MM 'URL-Type for parameters' should be set to 'Slug and Get allowed'

### ISSUE THREAD

**Issue by MacKP** [MetaModels/core#1456]
## Checklist before I submit this issue report

I confirm that:
- [ x ] I have tested this with the latest version available
- [ x ] I have read documentation @ http://metamodels.readthedocs.org/en/ or http://metamodels.readthedocs.org/de/
- [ x ] I have checked the Contao community forums for references https://community.contao.org/
- [ x ] I have checked existing issues for duplicates and found none @ https://github.com/MetaModels/core/issues?q=is%3Aissue

## My environment is:

(Please fill in the actual values from your environment - check Contao Manager or use `composer show`)

| Key                                 | Value    | Comments                         |
| ----------------------------------- | ---------| ---------------------------------|
| PHP version:                        |   7.4       |                                  |
| Contao version:                     |  4.9. 22       |                                  |
| MetaModels version:                 |   2.2.dev       |                                  |
| Installed MetaModels packages:      |          |                                  |
| DCG version:                        |          |                                  |

## Issue description

After updating from MM 2.1 to 2.2 i didn't get filter-Parameters in my Hook. The Problem was the Setting of 'URL-Type for parameters' in the Module. It was set to 'NULL' in the DB.
For BC these Setting should be set to 'Slug and Get allowed' via migration.
Or if there is 'NULL' set in DB it should work like both is selected?

Kind regards

**Comment by zonky2** [comment]
workaround

```UPDATE tl_content
SET metamodel_sort_param_type = 'slug'
WHERE metamodel_sort_param_type = ''
```
and
```
UPDATE tl_module
SET metamodel_sort_param_type = 'slug'
WHERE metamodel_sort_param_type = ''
```

**Comment by zonky2** [comment]
fixed in MM 2.2
